Extensions/ParticleSystem/particleemitterobject-pixi-renderer.js

```diff
@@ -0,0 +1,264 @@
+
+    this.renderer = new PIXI.Container();
+    this.emitter = new PIXI.particles.Emitter(this.renderer, texture, config);
+    this.emitter.emit = true;
+    this.started = false;
+
+    var layer = runtimeScene.getLayer("");
+    if (layer) layer.getRenderer().addRendererObject(this.renderer, runtimeObject.getZOrder());
+};
+gdjs.ParticleEmitterObjectRenderer = gdjs.ParticleEmitterObjectPixiRenderer;
+
+gdjs.ParticleEmitterObjectPixiRenderer.prototype.getRendererObject = function(){
+    return this.renderer;
+};
+
+gdjs.ParticleEmitterObjectPixiRenderer.prototype.update = function(delta){
+    this.emitter.update(delta);
+    if(!this.started && this.getParticleCount() > 0){
+        this.started = true;
+    }
+};
+
+gdjs.ParticleEmitterObjectPixiRenderer.prototype.setPosition = function(x, y){
+    this.emitter.spawnPos.x = x;
+    this.emitter.spawnPos.y = y;
+};
+
+gdjs.ParticleEmitterObjectPixiRenderer.prototype.setAngle = function(angle1, angle2){
+    this.emitter.minStartRotation = angle1;
+    this.emitter.maxStartRotation = angle2;
+};
+
+gdjs.ParticleEmitterObjectPixiRenderer.prototype.setForce = function(min, max){
+    // Small fix because different uses, instead random min and max speeds,
+    // it uses initial and final speeds
+    if(this.emitter.acceleration.x === 0 && this.emitter.acceleration.y === 0 && min === 0 && max !== 0){
+        min = 0.00001;
+    }
+
+    this.emitter.startSpeed.value = min;
+    if(this.emitter.startSpeed.next) this.emitter.startSpeed.next.value = max;
+};
+
+gdjs.ParticleEmitterObjectPixiRenderer.prototype.setZoneRadius = function(radius){
+    this.emitter.spawnCircle.radius = radius;
+};
+
+gdjs.ParticleEmitterObjectPixiRenderer.prototype.setLifeTime = function(min, max){
+    this.emitter.minLifetime = min;
+    this.emitter.maxLifetime = max;
+};
+
+gdjs.ParticleEmitterObjectPixiRenderer.prototype.setGravity = function(x, y){
+    this.emitter.acceleration.x = x;
+    this.emitter.acceleration.y = y;
+};
+
+gdjs.ParticleEmitterObjectPixiRenderer.prototype.setColor = function(r1, g1, b1, r2, g2, b2){
+    this.emitter.startColor.value.r = r1;
+    this.emitter.startColor.value.g = g1;
+    this.emitter.startColor.value.b = b1;
+    this.emitter.startColor.next.value.r = r2;
+    this.emitter.startColor.next.value.g = g2;
+    this.emitter.startColor.next.value.b = b2;
+};
+
+gdjs.ParticleEmitterObjectPixiRenderer.prototype.setSize = function(size1, size2){
+    this.emitter.startScale.value = size1/100.0;
+    if(this.emitter.startScale.next){
+        this.emitter.startScale.next.value = size2/100.0;
+    }
+};
+
+gdjs.ParticleEmitterObjectPixiRenderer.prototype.setAlpha = function(alpha1, alpha2){
+    this.emitter.startAlpha.value = alpha1/255.0;
+    if(this.emitter.startAlpha.next){
+        this.emitter.startAlpha.next.value = alpha2/255.0;
+    }
+};
+
+gdjs.ParticleEmitterObjectPixiRenderer.prototype.setFlow = function(flow){
+    this.emitter.frequency = 1.0/flow;
```

Comment:
If I pass 0 as the flow, is it still working properly? :) Just want to be sure that there is no error on pixi-particles side (as 1.0/0 in JavaScript = `Infinity`).

Reply:
Nice catch, I'll try that out :D
If I'm lucky, if frequency =infinite then it will take infinite time to create a new particle, it's flow = 0, but I've to check this one

Reply:
It works actually, anyway we have to let people to test the entire extension a lot, I've tried many options but I don't know, maybe a weird combination breaks it :)